inc/alert.class.php

```diff
@@ -52,7 +52,9 @@ class Alert extends CommonDBTM {
    const ACTION      = 5;
    const PERIODICITY = 6;
 
-
+   /**
+    * @see CommonDBTM::prepareInputForAdd($input)
```

Comment:
- [Project guideline](http://glpi-developer-documentation.readthedocs.io/en/master/codingstandards.html#functions)
![image](https://cloud.githubusercontent.com/assets/6187125/22179286/7678a918-e04f-11e6-82de-65138b657fa6.png)

I don't like when doc isn't up to date :-( (we all loose time on this)


Without @inheritdoc or @see, I loose inline doc on Netbeans (which use apigen :-D) for theses functions, but this isn't GLPI problem ;-)

Just to be sure : we delete docblock for overiden function when they don't give more precise information than parent docblock ?